lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -65,8 +67,8 @@
       <h2 class="hd hd-2 sr" id="details-heading-${enrollment.course_id}">${_('Course details')}</h2>
     <div class="wrapper-course-image" aria-hidden="true">
       % if show_courseware_link and not is_unfulfilled_entitlement:
-        % if not is_course_blocked:
-            <a href="${course_target}" data-course-key="${enrollment.course_id}" class="cover" tabindex="-1">
+        % if not is_course_blocked and not is_course_expired:
+            <a href="${course_target}" data-course-key="${enrollment.course_id}" class="cover course-target-link" tabindex="-1">
```

Comment:
Is this class just one you added for testing?

Reply:
yes